plugins/dovecot/dovecot

```diff
@@ -53,76 +78,52 @@ if [ "$1" = "config" ]; then
 	done
 
 	echo 'connected.label Connected Users'
-	echo "connected.type DERIVE"
+	echo "connected.type ABSOLUTE"
 
 	exit 0
 fi
 
-######################
-# Total Logins
-######################
-echo -en "login_total.value "
-VALUE=$(egrep -c '[dovecot]?.*Login' $LOGFILE)
-if [ ! -z "$VALUE" ]; then
-	echo "$VALUE"
+if [ "$LOGFILE" = "journald" -a "$TYPE" = "ABSOLUTE" ]; then
+        CURSOR_FILE="$MUNIN_STATEFILE"
+        # read cursor
+        # format: "journald-cursor <cursor>"
+        CURSOR=
+        if [ -f "$CURSOR_FILE" ]; then
+                CURSOR=$(awk '/^journald-cursor / {print $2}' "$CURSOR_FILE")
+        fi
 else
-	echo "0"
+        CURSOR_FILE=
 fi
-echo -n
+
+if [ "$LOGFILE" = "journald" ]; then
+        # shellcheck disable=SC2086
+        if [ "$TYPE" = "ABSOLUTE" ]; then
+                journalctl --no-pager --quiet --show-cursor ${CURSOR:+"--after-cursor=$CURSOR"} $JOURNALCTL_ARGS
+        else
+        journalctl --no-pager --quiet --since=$(date -dlast-sunday +%Y-%m-%d) $JOURNALCTL_ARGS
+        fi
+else
+        cat "$LOGFILE"
+fi | \
+    awk -v cursor_file="$CURSOR_FILE" -v connected="$(doveadm who -1 | expr $(wc -l) - 1)" 'BEGIN{c["login_total"]=0;c["connected"]=connected;c["login_tls"]=0;c["login_ssl"]=0;c["login_imap"]=0;c["login_pop3"]=0; }
```

Comment:
```bash
connected="$(doveadm who -1 | expr $(wc -l) - 1)"
```

Is this part correct? Piping to `expr`?

Reply:
Yes, it is. I sutract 1 from the number of lines (wc -l) because (coveadm who -1) contains one line of headers.

Reply:
Oh I see. I couldn't find any docs on what `expr` does with its standard input. I would usually do this with `tail`, like 

```bash
connected="$(doveadm who -1 | tail +2 | wc -l)"
```

Reply:
Your solution would have worked just as well. I simply used the more "arithmetic" approach.
Here is the man page of `expr`: https://man7.org/linux/man-pages/man1/expr.1.html
Thanks for merging.

Reply:
Oh, now I see your confusion. It could also have been
`connected="$(expr $(doveadm who -1 | wc -l) - 1)"`
would have even been more readable.